papers/doc/7-systems/ym2612.md

```diff
@@ -64,4 +66,4 @@ one of two chips that powered the Sega Genesis. It is a six-channel, four-operat
 - `5Cxx`: set D2R/SR of operator 1.
 - `5Dxx`: set D2R/SR of operator 2.
 - `5Exx`: set D2R/SR of operator 3.
-- `5Fxx`: set D2R/SR of operator 4.
\ No newline at end of file
+- `5Fxx`: set D2R/SR of operator 4.
```

Comment:
What's the change here?

Reply:
No difference here. No idea what bugged diff